common/app/common/commercial/hosted/HostedVideoPage.scala

```diff
@@ -82,6 +82,7 @@ object HostedVideoPage extends Logging {
           ),
           // todo: standardise css so that only colour varies
           cssClass = "renault",
```

Comment:
No, just leaving it there so that the page works before colour has been isolated in the css.